doc/prototype/is-defined-and-has-members.md

```diff
@@ -5,16 +5,24 @@ Return if a query can have records.
 
 @signature `queryLogic.isDefinedAndHasMembers(query)`
 
-Returns `true` if _query_ can have records. It returns `true` if the query is not
-[can-query-logic.EMPTY], [can-query-logic.UNDEFINABLE], or [can-query-logic.UNKNOWABLE].
+  Returns the query object if _query_ can have records. It returns `true` if the query is not
+  [can-query-logic.EMPTY], [can-query-logic.UNDEFINABLE], or [can-query-logic.UNKNOWABLE].
 
-```js
-import QueryLogic from "can-query-logic";
-var queryLogic = new QueryLogic();
-queryLogic.isDefinedAndHasMembers({}) //-> true
-queryLogic.isDefinedAndHasMembers(QueryLogic.UNIVERSAL) //-> true
-queryLogic.isDefinedAndHasMembers(QueryLogic.UNDEFINABLE) //-> false
-```
+  ```js
+  import {QueryLogic} from "can";
+  const queryLogic = new QueryLogic();
+
+  const emptyObject = queryLogic.isDefinedAndHasMembers({});
+  console.log( emptyObject ); //-> {}
+
+  const universal = queryLogic.isDefinedAndHasMembers(QueryLogic.UNIVERSAL);
+  console.log( universal ); //-> QueryLogic.UNIVERSAL
+
+  const undefinable = queryLogic.isDefinedAndHasMembers(QueryLogic.UNDEFINABLE); 
+  console.log( undefinable );//-> false
+  ```
+  @codepen
 
   @param  {can-query-logic/query} a A query.
-  @return {Boolean}
+  @return {can-query-logic/query|Boolean} returns `false` if _query_ is
```

Comment:
probably shouldn't be returning the query. Is it used this way in any of can-query-logic?  If not, we should probably make sure it always returns a boolean.

Reply:
I do not know of any example in can-query-logic where this is a benefit. Opened an issue -> https://github.com/canjs/can-query-logic/issues/22